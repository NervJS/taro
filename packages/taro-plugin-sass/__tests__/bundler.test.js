const path = require('path')
const getBundleContent = require('../bundler')

const filePath = path.resolve(__dirname, '.', 'styles/variables.scss')
const fileConfig = {
  name: 'styles/variables.scss',
  path: path.resolve(__dirname, '.')
}

describe('getBundleContent', () => {
  test('test file path', async () => {
    const res = await getBundleContent(filePath)
    expect(res.bundledContent).toMatchSnapshot()
  })

  test('test file name and project directory path', async () => {
    const res = await getBundleContent(fileConfig.name, fileConfig.path)
    expect(res.bundledContent).toMatchSnapshot()
  })
})
