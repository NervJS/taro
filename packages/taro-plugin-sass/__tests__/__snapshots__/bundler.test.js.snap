// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getBundleContent test file name and project directory path 1`] = `
"$color-ui-info: #78A4FA !default;
$color-ui-positive: #13CE66 !default;
$color-ui-negative: #FF4949 !default;
$color-ui-warning: #FFC82C !default;
"
`;

exports[`getBundleContent test file path 1`] = `
"$color-ui-info: #78A4FA !default;
$color-ui-positive: #13CE66 !default;
$color-ui-negative: #FF4949 !default;
$color-ui-warning: #FFC82C !default;
"
`;
